resources/rest-service/cloudify/migrations/versions/387fcd049efb_5_0_5_to_5_1.py

```diff
@@ -106,14 +106,12 @@ def _create_inter_deployment_dependencies_table():
                   nullable=False),
         sa.Column('dependency_creator', sa.Text(), nullable=False),
         sa.Column('target_deployment_func', JSONString(), nullable=True),
-        sa.Column('_source_deployment',
-                  sa.Integer(),
-                  nullable=False),
-        sa.Column('_target_deployment',
-                  sa.Integer(),
-                  nullable=True),
+        sa.Column('_source_deployment', sa.Integer(), nullable=True),
+        sa.Column('_target_deployment', sa.Integer(), nullable=True),
         sa.Column('_tenant_id', sa.Integer(), nullable=False),
         sa.Column('_creator_id', sa.Integer(), nullable=False),
+        sa.Column('external_source', JSONString(), nullable=True),
```

Comment:
I guess you also need to add the opposite logic to `remove`?

Reply:
don't need since we drop the entire table in remove